lnrpc/autopilotrpc/autopilot_server.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/grpc-ecosystem/grpc-gateway/runtime"
 	"github.com/lightningnetwork/lnd/autopilot"
 	"github.com/lightningnetwork/lnd/lnrpc"
-	"google.golang.org/grpc"
+	grpc "google.golang.org/grpc"
```

Comment:
Why is this necessary?

Reply:
Not sure, this was done automatically by the import tool. Reverted.